_inc/jetpack-jitm.js

```diff
@@ -176,6 +180,22 @@ jQuery( document ).ready( function( $ ) {
 				}, 2000 );
 			} );
 		} );
+
+		// Handle CTA ajax actions.
+		$template.find( '.jitm-button[data-ajax-action]' ).click( function( e ) {
+			e.preventDefault();
+			var button = $( this );
+			button.attr( 'disabled', true );
+			$.post( window.ajaxurl, {
+				action: button.data( 'ajax-action' ),
```

Comment:
Oh, I think this also needs a nonce to be checked on the server-side. There's an example in the script for D38263-code, based on a helpful article by our own @ebinnion : https://eric.blog/2013/06/18/how-to-add-a-wordpress-ajax-nonce/ 🎉

Reply:
Added in c48ffdb